spec/tictactoe/board_spec.rb

```diff
@@ -39,5 +39,13 @@
       board.positions = Array.new(9, "X")
       expect(board.is_not_full).to eq (false)
     end
+    it "should clear board"do
+    board = Tictactoe::Board.new
+    board.positions = Array.new(9, "X")
+    board.clear_board
+    expect(board.positions.count("-")).to be (9)
```

Comment:
It may be ok to use `expect(board.is_not_full).to eq (false)` here so that your test is not coupled with `-` string.